fastlane/lib/fastlane/setup/setup.rb

```diff
@@ -2,6 +2,11 @@ module Fastlane
   class Setup
     # Start the setup process
     def run
+      if FastlaneFolder.setup? and !Helper.is_test?
+        UI.important("fastlane already set up at path #{FastlaneFolder.path}")
```

Comment:
While we're here, a small improvement to grammar:

"fastlane already" => "fastlane is already"